parm/config/gefs/yaml/defaults.yaml

```diff
@@ -15,7 +15,20 @@ base:
   FCST_BREAKPOINTS: "48"
   REPLAY_ICS: "NO"
   USE_OCN_PERTURB_FILES: "false"
+  FHOUT_GFS: 6
   FHOUT_OCN_GFS: 6
   FHOUT_ICE_GFS: 6
   HPSSARCH: "NO"
   LOCALARCH: "NO"
+  SFS_POST: "NO"
+fcst:
+  reforecast: "YES"
```

Comment:
The default should be "NO" and turned on for the tests that need it.

Reply:
I changed the default to NO